client/webserver/site/src/html/forms.tmpl

```diff
@@ -67,11 +67,11 @@
       <label for="newWalletPass" class="form-label pt-3 ps-1 mb-1">[[[Wallet Password]]]
         <span class="ico-info" data-tooltip="[[[w_password_tooltip]]]"></span>
       </label>
-      <input type="password" class="form-control select" data-tmpl="newWalletPass" autocomplete="off">
+      <input type="password" class="form-control select" data-tmpl="newWalletPass" autocomplete="new-password">
```

Comment:
But this isn't a new password, e.g. it won't benefit from use of a password manager. I think `off` is the best choice here.